.github/workflows/build.yml

```diff
@@ -9,7 +9,8 @@ env:
   # This artifact is created in the 'validate-submission' job. It contains a JSON representation of
   # the Submision object. It can be read by other jobs to access those information rather than
   # fetch them from the GitHub API a second time.
-  SUBMISSION_PATH: "submission.json"
+  SUBMISSION_PATH: "${{ github.workspace }}/submission.json"
+  SUBMISSION_ARTIFACT_NAME: submission
```

Comment:
1.NIT: In the comment, `Submission` is misspelled. 
2. should SUBMISSION_ARTIFACT_NAME have a file extension? It's still a JSON file, no?
3. Should SUBMISSION_PATH use SUBMISSION_ARTIFACT_NAME in its definition?